### PR TITLE
PLF-6921 Move the code show/hide bottom toolbar into common project

### PR DESCRIPTION
**Pull request by nttuyen**

### CODE REVIEW

File: commons-extension-webapp/src/main/webapp/ckeditorCustom/config.js [hunk 1]
```diff
@@ -71,6 +72,9 @@ CKEDITOR.editorConfig = function( config ) {
 
     config.language = eXo.env.portal.language || 'en';
 
+    //This is config for hideBottomToolbar plugin
+    config.showBottomToolbar = true;
```
Comment (thomasdelhomenie) [inline commons-extension-webapp/src/main/webapp/ckeditorCustom/config.js]: Do we really need a configuration to activate it ? I mean, if I don't want to activate it, I can just remove the plugin from config.extraPlugins.

Comment (nttuyen) [inline commons-extension-webapp/src/main/webapp/ckeditorCustom/config.js]: Done, PR updated

File: commons-extension-webapp/src/main/webapp/eXoPlugins/hideBottomToolbar/plugin.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+require(['SHARED/jquery'],function($) {
```
Comment (thomasdelhomenie) [inline commons-extension-webapp/src/main/webapp/eXoPlugins/hideBottomToolbar/plugin.js]: Can you add a global comment on this component to explain what it does please ?
Also I think we should mention that it is dependent on some CSS classes (to be listed).

Comment (nttuyen) [inline commons-extension-webapp/src/main/webapp/eXoPlugins/hideBottomToolbar/plugin.js]: Updated.

**Pull request merged**